frontend/src/components/pages/teacher/DisplayAssessmentResultsPage/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Flex, IconButton, Text } from "@chakra-ui/react";
+
+import { LeftArrowIcon } from "../../../../assets/icons";
+import * as Routes from "../../../../constants/Routes";
+import RouterTabs from "../../../common/RouterTabs";
+import NotFound from "../../NotFound";
+
+import DisplayAssessmentResultsByStudentPage from "./DisplayAssessmentResultsByStudentPage";
+import DisplayAssessmentResultsSummaryPage from "./DisplayAssessmentResultsSummaryPage";
+
+const TAB_CONFIG = [
+  {
+    name: "Summary",
+    path: Routes.DISPLAY_ASSESSMENT_RESULTS_SUMMARY_PAGE,
+    component: DisplayAssessmentResultsSummaryPage,
+  },
+  {
+    name: "Student",
+    path: Routes.DISPLAY_ASSESSMENT_RESULTS_BY_STUDENT_PAGE,
+    component: DisplayAssessmentResultsByStudentPage,
+  },
+  {
+    path: "*",
+    component: NotFound,
+  },
+];
+
+const DisplayAssessmentResults = () => {
+  const assessmentName = "Grade 5 Ontario Pre-Term Assessment";
+  return (
+    <Flex direction="column" gap={8}>
+      <Text as="h1" color="blue.300" textStyle="header4">
+        <IconButton
+          aria-label="Back"
+          icon={<LeftArrowIcon h={8} w={8} />}
+          minW={0}
+          mr={6}
+        />
```

Comment:
nit: there's an existing `BackButton` component that you might be able to adapt? but if it's not suitable for this, as is is fine too. but if we leave it as is, can we add hover behaviour that matches our other back buttons? 🙏

Reply:
I didn't like the existing back button because it navigates back via `history.goBack()` as opposed to actually going to the page the user came from. But if you think that behaviour is fine, I'm fine with that. Would just have to change the `RouterTabs` component to use `replace` rather than `push`.